### ISSUE TITLE
Reach 100% coverage of Python code

### ISSUE THREAD

**Issue by godlygeek** [bloomberg/pystack#51]
Our `make pycoverage` command currently has `--cov-fail-under=97` to fail the test suite if the Python code coverage level drops below 80%. Let's get that up to 100% (adding a `# pragma: no cover` for anything that's not reasonably testable), and update this to `--cov-fail-under=100`.

**Comment by yize415** [comment]
I can work on this one!

**Comment by gusmonod** [comment]
This is technically already done, although after #86 the python code coverage will also include Cython coverage, which will move it back to ~96% or so. Still, all of the Python code is technically covered, so I'm closing this issue and adding one for Cython: #88

**Comment by gusmonod** [comment]
Thank you @yize415!
